translations/default/AdminActions.xlf

```diff
@@ -814,546 +922,41 @@
       </trans-unit>
     </body>
   </file>
-  <file original="src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Actions/Bulk/submit.html.twig" source-language="en-US" target-language="en" datatype="plaintext">
-    <body>
-      <trans-unit id="70d9be9b139893aa6c69b5e77e614311">
-        <source>Confirm</source>
-        <target>Confirm</target>
```

Comment:
I'm not sure to understand why these translations are removed, they seem to still be present 🤔

Reply:
I don't know either it's weird the wording is still in the twig file though... That's a bit worrying, it would mean that we don't scan twig files correctly anymore? 🤔
I rebased the branch and I'm performing a new extract to see if it happens again.

Reply:
So the problem was because of a bug on our visitor that is used to extract wording, it's been fixed in this PR https://github.com/PrestaShop/TranslationToolsBundle/pull/107